soundata/datasets/tau2020uas_mobile.py

```diff
@@ -3,32 +3,31 @@
 .. admonition:: Dataset Info
     :class: dropdown
 
-    **TAU Urban Acoustic Scenes 2020 Mobile, Development and Evaluation
-    datasets**
+    **TAU Urban Acoustic Scenes 2020 Mobile, Development and Evaluation datasets**
 
     `Audio Research Group,
-    Tampere University of Technology <http://arg.cs.tut.fi/>`_
+    Tampere University of Technology <http://arg.cs.tut.fi/>`__
```

Comment:
2 whitespaces is because of the warnings, right? (duplicated comment)

Reply:
Yes... was having a lot of problems with hyperlinks (especially those internal), and using double underscore supressed them all. It still works fine, right?

Reply:
It does, I guess we can keep it like this